CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h

```diff
@@ -257,6 +258,34 @@ namespace SiStripPI {
       
     }
   }
+
+
+  
+  // code is mutuated from CalibTracker/SiStripQuality/plugins/SiStripQualityStatistics
+
+  /*--------------------------------------------------------------------*/
+  void setBadComponents(int i, int component, SiStripQuality::BadComponent& BC,int NBadComponent[4][19][4])
+  /*--------------------------------------------------------------------*/
+  {
+   
+    if (BC.BadApvs){
+      NBadComponent[i][0][2]+= ( (BC.BadApvs>>5)&0x1 )+ ( (BC.BadApvs>>4)&0x1 ) + ( (BC.BadApvs>>3)&0x1 ) + 
```

Comment:
hi @mmusich - can this be written as 
+=std::bitset<16>(BC.BadApvs&03F).count()
(16 depends on the type for BadApvs)
?